pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -184,10 +188,10 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, createValidati
 	}
 
 	// Stop on the first error, since we have to delete the whole project if any item in the template fails
-	stopOnErr := configcmd.AfterFunc(func(_ *resource.Info, err error) bool {
-		// if something already exists, we're probably racing a controller.  Don't die
-		if kapierror.IsAlreadyExists(err) {
-			return false
+	stopOnErr := configcmd.AfterFunc(func(info *resource.Info, err error) bool {
+		// if a default role binding already exists or is being created, we're probably racing the controller.  Don't die
+		if defaultRoleBindingNames.Has(info.Name) && info.Mapping.GroupVersionKind.GroupKind() == roleBindingKind {
+			return utilerrors.FilterOut(err, kapierror.IsAlreadyExists, kapierror.IsConflict) != nil
```

Comment:
you'll never have a conflict on a create

Reply:
Ah misread the failure cause in the logs, will remove.